products/iambeta/terraform.yaml

```diff
@@ -32,13 +32,56 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         min_version: beta
     docs: !ruby/object:Provider::Terraform::Docs
       attributes: |
-        * `self_link`: The self link of the created WorkloadIdentityPool in the format `projects/{project}/locations/global/workloadIdentityPools/{workload_identity_pool_id}`
+        * `self_link`: The self link of the created WorkloadIdentityPool in the format
+          `projects/{project_number}/locations/global/workloadIdentityPools/{workload_identity_pool_id}`
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       constants: templates/terraform/constants/iam_workload_identity_pool.go.erb
     properties:
       workloadIdentityPoolId: !ruby/object:Overrides::Terraform::PropertyOverride
         validation: !ruby/object:Provider::Terraform::Validation
           function: 'validateWorkloadIdentityPoolId'
+  WorkloadIdentityPoolProvider: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
+    import_format: ["projects/{{project}}/locations/global/workloadIdentityPools/{{workload_identity_pool_id}}/providers/{{workload_identity_pool_provider_id}}"]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "iam_workload_identity_pool_provider_aws_basic"
+        primary_resource_id: "example"
+        vars:
+          workload_identity_pool_id: "example-pool"
+          workload_identity_pool_provider_id: "example-prvdr"
+        min_version: beta
+      - !ruby/object:Provider::Terraform::Examples
+        name: "iam_workload_identity_pool_provider_aws_full"
+        primary_resource_id: "example"
+        vars:
+          workload_identity_pool_id: "example-pool"
+          workload_identity_pool_provider_id: "example-prvdr"
+        min_version: beta
+      - !ruby/object:Provider::Terraform::Examples
+        name: "iam_workload_identity_pool_provider_oidc_basic"
+        primary_resource_id: "example"
+        vars:
+          workload_identity_pool_id: "example-pool"
+          workload_identity_pool_provider_id: "example-prvdr"
+        min_version: beta
+      - !ruby/object:Provider::Terraform::Examples
+        name: "iam_workload_identity_pool_provider_oidc_full"
+        primary_resource_id: "example"
+        vars:
+          workload_identity_pool_id: "example-pool"
+          workload_identity_pool_provider_id: "example-prvdr"
+        min_version: beta
+    docs: !ruby/object:Provider::Terraform::Docs
+      attributes: |
+        * `self_link`: The self link of the created WorkloadIdentityPoolProvider in the format
```

Comment:
Hmm- I don't think this field actually exists, it isn't in the schema: [`auto-pr-4129-old..auto-pr-4129`#diff-40bc55eff6](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-4129-old..auto-pr-4129#diff-40bc55eff6e654bd6c6a035d43ed4c31f826fd2b370875f6e66f447f50f8f57cR73)

I think we can just remove this docs entry? The `id` field will actually have this same value and should already be documented.

Reply:
Correct, this was copy-paste work from other resources and these workload identity resources have no self_link. I removed this from both resources